src/test/groovy/com/gooddata/executeafm/afm/OverPeriodMeasureDefinitionTest.groovy

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2007-2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.executeafm.afm
+
+import com.gooddata.executeafm.IdentifierObjQualifier
+import com.gooddata.executeafm.UriObjQualifier
+import nl.jqno.equalsverifier.EqualsVerifier
+import org.apache.commons.lang3.SerializationUtils
+import spock.lang.Specification
+
+import static com.gooddata.util.ResourceUtils.readObjectFromResource
+import static net.javacrumbs.jsonunit.JsonMatchers.jsonEquals
+import static net.javacrumbs.jsonunit.core.util.ResourceUtils.resource
+import static spock.util.matcher.HamcrestSupport.that
+
+class OverPeriodMeasureDefinitionTest extends Specification {
+
+    private static final String path = 'executeafm/afm'
+    private static final String SINGLE_ATTRIBUTES_JSON = "$path/overPeriodMeasureDefinition.json"
```

Comment:
isn't it `SINGLE_ATTRIBUTE...` (without `S`)?

Reply:
Fixed.